phpunit/class-wp-theme-json-test.php

```diff
@@ -567,109 +567,6 @@ function test_remove_insecure_properties_removes_unsafe_styles() {
 		$this->assertEqualSetsWithIndex( $expected, $actual );
 	}
 
-	function test_remove_insecure_properties_removes_unsafe_styles_sub_properties() {
-		$theme_json = new WP_Theme_JSON(
-			array(
-				'version' => WP_Theme_JSON::LATEST_SCHEMA,
-				'styles'  => array(
-					'spacing'  => array(
-						'padding' => array(
-							'top'    => '1px',
-							'right'  => '1px',
-							'bottom' => 'var(--unsafe-var-toplevel)',
```

Comment:
Maybe instead we should test for something else.

Reply:
yup https://github.com/WordPress/gutenberg/pull/31982/files#diff-88c997ea8e8ba51f2fa201327e0f1faccd4e892c2fafe33249f72cfd068c72fcR579